package.json

```diff
@@ -288,7 +289,8 @@
     "@bithighlander/bitcoin-cash-js-lib@^5.2.1": "patch:@bithighlander/bitcoin-cash-js-lib@npm%3A5.2.1#./.yarn/patches/@bithighlander-bitcoin-cash-js-lib-npm-5.2.1-92e8f8436e.patch",
     "friendly-challenge@0.9.2": "patch:friendly-challenge@npm%3A0.9.2#./.yarn/patches/friendly-challenge-npm-0.9.2-db0cad46d3.patch",
     "react-dom@^18.2.0": "patch:react-dom@npm%3A18.2.0#./.yarn/patches/react-dom-npm-18.2.0-dd675bca1c.patch",
-    "react-dev-utils/fork-ts-checker-webpack-plugin": "^6.5.3"
+    "react-dev-utils/fork-ts-checker-webpack-plugin": "^6.5.3",
+    "p-queue": "^6.6.2"
```

Comment:
MM snaps uses p-queue v7+, which is a ["Pure ESM"](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) package.
Since it doesn't define a main field in its package.json, jest is unable to import it, and no amount of fiddling with `transformIgnorePatterns` nor `moduleNameMapper` solved things here, this is inherent to the package module system.

Another solution could be to use https://github.com/esm2cjs/p-queue in snaps, though using v6 resolution seems like a safer option vs. a fork, despite it being an older version, see:

https://github.com/sindresorhus/p-queue/blob/345e55352b0f1b3a3c8d4e1d511a8ccd5d9510a1/readme.md

> You can also use [version 6](https://github.com/sindresorhus/p-queue/tree/v6.6.2) instead which is pretty stable. We will backport security fixes to v6 for the foreseeable future.

Since 7.0.0 doesn't bring any breaking changes that make the two version incompatible, decided to keep the snap adapter version to latest, but use a resolution to latest v6 here.